docker-compose.yml

```diff
@@ -8,6 +8,16 @@ services:
       - .:/usr/chapter
     ports:
       - '8000:8000'
+  db:
+    image: postgres
+    environment:
+      - POSTGRES_USER=root
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_DB=chapter
+    volumes:
```

Comment:
Same as above - not necessary to persist the psql data (wouldn't need a volume in this case) - let me know your thoughts